chain/chain.go

```diff
@@ -68,7 +68,7 @@ func (g *Genesis) GenesisHeader() *types.Header {
 		GasUsed:      g.GasUsed,
 		Difficulty:   g.Difficulty,
 		MixHash:      g.Mixhash,
-		Miner:        g.Coinbase,
+		Miner:        g.Coinbase[:],
```

Comment:
Why was this necessary?

Reply:
Coinbase: Address Type
Miner field: []byte Type

Reply:
Because of type difference